src/realizations/catchment/CMakeLists.txt

```diff
@@ -27,9 +26,13 @@ target_link_libraries(realizations_catchment PUBLIC
         NGen::models_tshirt
         NGen::geojson
         NGen::kernels_evapotranspiration
-        pybind11::embed
         )
 
+if(NGEN_ACTIVATE_PYTHON)
+   target_include_directories(realizations_catchment PUBLIC ${PROJECT_SOURCE_DIR}/extern/pybind11/include)
+   target_link_libraries(realizations_catchment PUBLIC pybind11::embed)
+endif()
+
```

Comment:
I think the changes in this file in particular should now be in the upstream repo.  You may want to rebase and re-push your branch.

Reply:
Agree.